tests/testthat/test-store.R

```diff
@@ -17,3 +17,66 @@ test_that("file store", {
   st$del(file)
   expect_false(st$exists(file))
 })
+
+test_that("caching file store", {
```

Comment:
Yeah, that would be better style.  It can get a bit tedious though as there's so much side effect heavy bits here (remake is basically _all_ side effect)